sourcemash/users/models.py

```diff
@@ -8,7 +8,14 @@
 
 role_users = db.Table('roles_users',
         db.Column('user_id', db.Integer, db.ForeignKey('user.id')),
-        db.Column('role_id', db.Integer, db.ForeignKey('role.id')))
+        db.Column('role_id', db.Integer, db.ForeignKey('role.id'))
+)
+
+users_items = db.Table('users_items',
```

Comment:
should this be a db.Assocation?
http://docs.sqlalchemy.org/en/rel_0_9/orm/basic_relationships.html#many-to-many